### ISSUE TITLE
Improvement: Create pushbutton to send confirmation email for an individual registration

### ISSUE THREAD

**Issue by arborrow** [arborrow/montserrat#410]
There are a couple of use cases:
1) Someone who has registered for a retreat after the automated, pre-retreat confirmation has been sent out
2) A non-Ignatian event where we want to send a pre-retreat confirmation 
3) It may be worthwhile to explore adding an option to store html code for each event for a customized pre-retreat confirmation email as well as to customize when that email goes out and display on the event.show blade when that confirmation is scheduled to go out

In terms of implementation, I am thinking that the pushbutton would best be located either on registration.show and/or retreat.show in the registration section. I would suggest, even though it might slow down the work process by an additional click to confirm sending of the email. When the email is sent, it should also add a touchpoint from the logged in user documenting that the email was sent. The email should be sent from the Polanco email account that would provide the history of exactly what was sent.

**Comment by arborrow** [comment]
I have added a Send confirmation email button on the registration.show blade that will display when the registration_confirm_date is blank. It doesn't make sense to send the email if the retreatant has already confirmed. The button assumes that the email confirmation snippets are up to date. I opted not to confirm sending of the email as it is isolated on the registration.show blade so less likely to be clicked by accident.
